packages/cli/src/cli/cogify/imagery.config.ts

```diff
@@ -0,0 +1,46 @@
+import { Config, ConfigImagery, ConfigTileSet, TileSetType } from '@basemaps/config';
+import { ImageFormat } from '@basemaps/geo';
+import { LogType } from '@basemaps/shared';
+import { CogStacJob } from '../../cog/cog.stac.job';
+
+/**
+ * Prepare and insert Imagery Config for the cog creation job.
+ * @returns
+ */
+export async function insertConfigImagery(job: CogStacJob, logger: LogType): Promise<void> {
+  const now = Date.now();
+  const imgId = Config.Imagery.id(job.id);
+  const configImagery: ConfigImagery = {
+    id: imgId,
+    name: job.name,
+    createdAt: now,
+    updatedAt: now,
+    projection: job.tileMatrix.projection.code,
```

Comment:
we should add tileMatrix identifier here too.

Reply:
Updated. We might need to update `basemap-config` repo, next time when we upgrade config package versions.

Reply:
Build failure but otherwise :lgtm: